keras_cv/layers/preprocessing/jittered_resize.py

```diff
@@ -206,24 +206,24 @@ def augment_bounding_boxes(self, bounding_boxes, transformation, **kwargs):
         )
 
         # Adjusts box coordinates based on image_scale and offset.
-        yxyx = bounding_boxes[:, :4]
-        rest = bounding_boxes[:, 4:]
+        yxyx = bounding_boxes["boxes"]
         yxyx *= tf.tile(tf.expand_dims(image_scale, axis=0), [1, 2])
         yxyx -= tf.tile(tf.expand_dims(offset, axis=0), [1, 2])
 
-        bounding_boxes = tf.concat([yxyx, rest], axis=-1)
-        bounding_boxes = keras_cv.bounding_box.clip_to_image(
-            bounding_boxes,
+        result["boxes"] = yxyx
+        result = keras_cv.bounding_box.clip_to_image(
+            result,
             image_shape=self.target_size + (3,),
             bounding_box_format="yxyx",
         )
-        bounding_boxes = keras_cv.bounding_box.filter_sentinels(bounding_boxes)
-        return keras_cv.bounding_box.convert_format(
-            bounding_boxes,
+        result = keras_cv.bounding_box.convert_format(
+            result,
             image_shape=self.target_size + (3,),
             source="yxyx",
             target=self.bounding_box_format,
         )
+        result = keras_cv.bounding_box.to_ragged(result, dtype=self.compute_dtype)
```

Comment:
BaseAugmentationLayer already runs `to_ragged` during `_augment`, why do we need to run it again here?

Reply:
Removed, correct!